apcd-cms/src/apps/view_users/views.py

```diff
@@ -30,29 +31,46 @@ def _set_user(usr):
                     'created_at': usr[5],
                     'updated_at': usr[6],
                     'notes': usr[7],
-                    'active': 'Active' if usr[8] else 'Inactive',
+                    'status': 'Active' if usr[8] else 'Inactive',
                     'user_number': usr[9],
                     'role_name': usr[10],
                     'org_name_no_parens': usr[4].replace("(", "").replace(")", ""),  # just for filtering purposes
+                    'active': 'Yes' if usr[8] else 'No', # just to tell active from inactive status
```

Comment:
I'm unsure if this necessary, I think you could use the `status` field and display Active or Inactive in the filter instead. I'm curious what others think!

Reply:
Yeh. I did what you suggested first but, the filter couldn't differentiate between 'ACTIVE' and 'inACTIVE' because it kept matcing.. ACTIVE

Reply:
Ah that makes sense - that seems buggy for a table filter. I'll add a change for that.